content-dev/src/main/content/jcr_root/content/phonegap/app_name_placeholder/shell/_jcr_content/pge-app/app-content/phonegap/www/index.html

```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8">
         <meta http-equiv="refresh" content="1;url=content/phonegap/app_name_placeholder/en.html">
         <script type="text/javascript">
-            window.location.href = "content/phonegap/app_name_placeholder/en.html";
+            window.location.href = "../../../../../../../..//phonegap/app_name_placeholder/en.html";
```

Comment:
This doesn't look correct to me. This should be a relative path to the splash page (as above in `meta http-equiv="refresh"` tag). The double `//` also looks off.